zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -32,10 +33,18 @@
 
 public final class Repository implements AutoCloseable {
 
-    static final String KEYSPACE = "zipkin";
-
-    private static final String KEY_MARKER = "k";
-    private static final String COLUMN_MARKER = "c";
+    public static final String KEYSPACE = "zipkin";
+
+    private static final String TRACE_ID = "trace_id";
```

Comment:
I've found most project avoid declaring constants just to snake case words in the same file. Seems the only below that isn't just a lower-snake is ts, but we use ts often in the codebase. By removing these, the file is less arduous to navigate.